app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -10,15 +11,20 @@ class RedirectIfAuthenticated
     /**
      * Handle an incoming request.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \Closure  $next
-     * @param  string|null  $guard
+     * @param \Illuminate\Http\Request $request
+     * @param Closure $next
+     * @param string[]|null ...$guards
+     *
```

Comment:
Let's revert these docblock changes for now.

Reply:
Do you also want me to revert the changes to the handle method?

Reply:
No it's fine.

Reply:
I guess I'm being slow, but in that case, why revert the docblock? This matches the Laravel source

Reply:
Indentation is off: https://github.com/laravel/laravel/blob/8.x/app/Http/Middleware/RedirectIfAuthenticated.php#L15-L17